triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -272,6 +284,20 @@ trait AbstractTriggerServiceTestHelper
     eventually {
       pred(getTriggerStatus(triggerInstance).map(_._2))
     }
+
+  def assertTriggerRunnerStatus(
+      triggerInstance: UUID,
+      pred: Vector[String] => Assertion,
+  ): Assertion = {
+    val filterString = s"id: \"$triggerInstance\""
+
+    eventually {
+      pred(triggerRunnerLogAppender.list.toArray.toVector.collect {
+        case event: ILoggingEvent if event.getMarker.toString contains filterString =>
+          event.toString
+      })
```

Comment:
We filter the logging events and apply our predicate to the set which have the given trigger ID - i.e. the predicate is applied to only log events emitted by the trigger runner with the given UUID.